lib/pages/AthletePage.dart

```diff
@@ -45,12 +53,69 @@ class _AthletePageState extends State<AthletePage> {
   Widget build(BuildContext context) {
     if (listView == 1) return DesktopScout();
 
-    return kIsWeb.kIsWeb ? buildWebView(context) : buildMobileView(context);
+    return kIsWeb.kIsWeb ? buildWebViewContainer(context) : buildMobileView(context);
+  }
+
+  IndexedStack buildWebViewContainer(BuildContext context) {
+
+    var longMarketPrice = "4.18 AX";
```

Comment:
make all these vars finals if they arent gonna be changing

Reply:
done